saleor/graphql/checkout/mutations/utils.py

```diff
@@ -54,6 +56,15 @@ class CheckoutLineData:
     metadata_list: Optional[list] = None
 
 
+def _is_external_shipping_valid(checkout_info: "CheckoutInfo"):
```

Comment:
Could we add this method below `update_checkout_external_shipping_method_if_invalid`? It will be easier to read, than jumping function above